target/linux/ramips/image/mt76x8.mk

```diff
@@ -100,6 +100,14 @@ define Device/d-team_pbr-d1
 endef
 TARGET_DEVICES += d-team_pbr-d1
 
+define Device/dlink_dap-1325-a1
+  IMAGE_SIZE := 7872k
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DAP-1325 A1
+  SUPPORTED_DEVICES += dap-1325-a1
```

Comment:
this line can be dropped, unless you need for factory flashing ...

Reply:
fixed